### ISSUE TITLE
Wrong duration detected for all YT videos

### ISSUE THREAD

**Issue by alfonder** [umputun/feed-master#113]
Metadata for all videos on any channel indicates duration 0s. Here is the log quote:
`feed                   | 2023/02/21 22:11:00.689 [INFO]  new entry [2] KXF7mN6fOSk, Is fecit cui prodest. Бенефициары и группы интересов. Разговор с Юлией Латыниной, Екатерина Шульман, {ChannelID:UCL1rJ0ROIw9V1qFeIN0ZTZQ, VideoID:KXF7mN6fOSk, Title:"Is fecit cui prodest. Бенефициары и группы интересов. Разговор с Юлией Латыниной", Published:2023-02-18T17:20:18+01:00, Updated:2023-02-19T08:32:30+01:00, Author:Екатерина Шульман, File:, Duration:0s}`
This issue prevents using "skip_shorts" option.
Tested on image https://hub.docker.com/r/umputun/feed-master (tag :master)

**Comment by umputun** [comment]
The "new entry" message is slightly misleading, but this is just it - a message. When this message was printed, the system had yet to have duration info available. If you let it work a little bit, you will see another message with a filled duration.

I don't see any actual problem with the system not filtering our shorts; it works perfectly fine in my case; see, for example, https://echo.feed-master.com, which is filtering `skip_shorts: 5m` for real and nothing shorter collected.

You can set `--dbg` flag (`DEBUG=true`) in compose to see more details of what skip_shorts detected exactly

**Comment by alfonder** [comment]
Hmm, it's interesting. In your feed there are items (from Plushev channel) which skipped on my instance. I waited for more than 24 hours and see only "new entry" and "skipping" log records. Unfortunately, it doesn't indicate why it skips them. 
Thanks for suggestion, I'll try with debug enabled.

**Comment by umputun** [comment]
Yeah, this is unlikely to have anything to do with skip_shorts; those are reported without debug like this

` [INFO]  skipping {ChannelID:UC0OiowQ-fTqw3Kq3bfopO_Q, VideoID:mmJqFbE2mTk, Title:"Андрей Колесников / Год войны/ Послание Путина/ Байден в Киеве // 21.2.23", Published:2023-02-21T01:23:55-06:00, Updated:2023-02`

**Comment by alfonder** [comment]
Running during a day with debug enabled didn't help. Somehow all videos are skipped. A part of log is in the attachment.
PS: thanks for mention in Radio-T.
[f-m-1.log](https://github.com/umputun/feed-master/files/10808594/f-m-1.log)

**Comment by umputun** [comment]
_I'll move it to discussions._

Having more details will help, namely yml configuration and docker compose
